[PLAYER-4647] Update *all* sample apps to support refactored OoyalaSDK

maxkupetskii opened the pull request:
# All sample apps are now *refactored* and cleaned out of deprecated/outdated methods/settings

While working on PLAYER-4647 I had to do a small refactor and because of that I had to update _all_ sample apps. Eventually it led to sort of standardizing of every app

#### Briefly
All sample apps now have the same build settings, default settings provided if the same, broken links removed, .dylib libs changed to .tbd

## Changes
### Added
* *.xcassets* were put back in every project

### Changed
* build settings
* unnecessary settings removed
* recommended settings applied
* *BasicPlaybackSampleApp* now has 2 audio only assets to test. After fixes on SAS there more will be added. *AudioOnlyPlayerInfo* added 

### Removed
* broken links
* incorrect settings
* signing from all projects

Commit:
[PLAYER-4355] Swift version bumped to 4.2

Commit:
[PLAYER-4355] Updates for Swift 4.2

-code style applied

Commit:
Merge branch 'dev' into PLAYER-4355

Commit:
[PLAYER-4647] AudioOnlyPlayerInfo added

-project updated

Commit:
[PLAYER-4647] BasicPlaybackListViewController updated

Commit:
[PLAYER-4647] PlayerSelectionOption updated

Commit:
[PLAYER-4647] Sample VC updated

Commit:
[PLAYER-4647] BasicSimplePlayerViewController updated

Commit:
[PLAYER-4647] project settings updated

Commit:
[PLAYER-4647] .xcschemes updated for all sample apps

Commit:
[PLAYER-4647] All sample apps updated

-recommended settings
- -0bjC flag
- .dylib changed to .tbd
- conflicting parameters removed
- code updated where necessary

Commit:
[PLAYER-4647] Projects updated

Commit:
[PLAYER-4647] Project settings updated

-assets fixed
-options reseted to default if the same

Commit:
[PLAYER-4647] Projects update

Commit:
[PLAYER-4647] PlaybackLab project updates

Commit:
Merge branch 'candidate' into PLAYER-4647

# Conflicts:
#	PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp.xcodeproj/project.pbxproj
#	PulseSampleApp/PulseSampleApp.xcodeproj/project.pbxproj

Commit:
[PLAYER-4647] DTO sample app fix

Commit:
[PLAYER-4647] LaunchImage to chromecast sample app added

Commit:
[PLAYER-4647] Fix for images

Commit:
[PLAYER-4647] Fixes for Chromecast sample app

Commit:
[PLAYER-4647] Dash Live channel updated

MarkEPAM commented on BasicPlaybackSampleApp/BasicPlaybackSampleApp/Utils/AudioOnlyPlayerInfo.h:
```diff
@@ -0,0 +1,13 @@
+//  AudioOnlyPlayerInfo.h
+//  BasicPlaybackSampleApp
+//
+//  Created by Max Kupetskii on 11/2/18.
```
remove your personal info

MarkEPAM commented on BasicPlaybackSampleApp/BasicPlaybackSampleApp/Utils/AudioOnlyPlayerInfo.m:
```diff
@@ -0,0 +1,17 @@
+//  AudioOnlyPlayerInfo.m
+//  BasicPlaybackSampleApp
+//
+//  Created by Max Kupetskii on 11/2/18.
```
remove your personal info

Commit:
[PLAYER-4647] Removed personal info

Commit:
[PLAYER-4647] Skin sample app fixes

Commit:
[PLAYER-4826] Options Sample app updated

Commit:
Merge branch 'dev' into PLAYER-4647

# Conflicts:
#	OoyalaSkinSampleApp/OoyalaSkinSampleApp.xcodeproj/project.pbxproj

Commit:
Merge branch 'PLAYER-4826' into PLAYER-4647

Commit:
[PLAYER-4647] PR updates

Commit:
Merge branch 'dev' into PLAYER-4355

Commit:
[PLAYER-4647] Fixes

Commit:
[PLAYER-4647] Swift VR app updated

# Conflicts:
#	SwiftVRSkinSampleApp/SwiftVRSkinSampleApp.xcodeproj/project.pbxproj

Commit:
[PLAYER-4647] Updates for apps

-automatic signs
-hearbeat swift app to 4.2

Commit:
[PLAYER-4647] - Updates

Commit:
[PLAYER-4647] Complete sample app fixes

Commit:
[PLAYER-4647] Removed apple watch sample app

-outdated
-not used

Commit:
[PLAYER-4647] Playback lab fixes

MarkEPAM reviewed (approved):
LGTM
